apps/core-create-react-app/src/index.tsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
 
+import 'normalize.css/normalize.css';
+import '@clr/core/global.css'; // pre-minified version breaks
```

Comment:
Not sure why but the version of postcss create react app uses does not like our pre-minified version of the global CSS.